docs/troubleshooting.md

```
@@ -0,0 +1,352 @@
+- [Browser Cache and State](#browser-cache-and-state)
+- [Debugging Tips](#debugging-tips)
+- [FAQ](#faq)
+
+---
+
+## Common Errors
+
+### Insufficient XLM for Fee
+
+**Error Message:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)